app/assets/stylesheets/forms.scss

```diff
@@ -1,3 +1,14 @@
 .govuk-label {
   font-weight: bold;
 }
+
+div#votes p  {
```

Comment:
You should probably create classes for this. An ID is for one HTML element only, there should never be multiple CSS elements with the same ID.